packages/core/src/extension/extension-service.ts

```diff
@@ -33,23 +33,19 @@ export default class ExtensionService {
         this._extensionId = extensionId;
 
         this._eventListener.listen();
+        this._commandPoster.setContext({ extensionId });
+        this._internalCommandPoster.post({
+            type: ExtensionInternalCommandType.ResizeIframe,
```

Comment:
Are we changing the `FRAME_LOADED` command to `ResizeIframe`?

Reply:
Yes, because `FRAME_LOADED` is an event, not a command. Therefore I renamed it to be a command.

Reply:
I think we just need to check if we were making listening to FrameLoaded specifically on checkout side, @bc-peng do you remember?

Reply:
If my understanding is correct, Checkout doesn't listen for it. Instead, it's listened by `ResizeableIframeCreator`.

Reply:
https://github.com/bigcommerce/checkout-sdk-js/blob/01378de2d1e69efa38214d863178457943eead2b/packages/core/src/embedded-checkout/resizable-iframe-creator.ts#L68
We display an iframe upon receiving it.